gluten-core/src/main/scala/io/glutenproject/extension/columnar/TransformHintRule.scala

```diff
@@ -593,18 +534,13 @@ case class AddTransformHintRule() extends Rule[SparkPlan] {
 
             val maybeExchange = buildSidePlan.find {
               case BroadcastExchangeExec(_, _) => true
-              case _ =>
-                printValidateLog(this.getClass.toString,
-                  "not BroadcastExchangeExec at join build side.")
-                false
+              case _ => false
```

Comment:
So this failure info will be lost?

Reply:
no, the `maybeExchange` is not the final validation result. The code here is used to find a broadcast exchange, and we would match both Some and None case to do the final validation, see line 576.